webservices/args.py

```diff
@@ -230,6 +230,7 @@ def make_seek_args(field=fields.Int, description=None):
     'party': fields.List(IStr, description='Three-letter code for the party. For example: DEM=Democrat REP=Republican'),
     'min_first_file_date': fields.Date(description='Minimum date of the first form filed by the committee.'),
     'max_first_file_date': fields.Date(description='Maximum date of the first form filed by the committee.'),
+    'treasurer': fields.Str(description='Committee treasurer'),
```

Comment:
One small thing. The variable name is `treasurer_name`. I think it is easier when the field names and the filter names are the same. Would that be ok?